easybuild/easyconfigs/z/zlib-ng/zlib-ng-2.0.5-GCCcore-10.2.0.eb

```diff
@@ -0,0 +1,30 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+# Updated: Denis Kristak
+easyblock = 'CMakeMake'
+
+name = 'zlib-ng'
+version = '2.0.5'
+
+homepage = 'https://github.com/zlib-ng/zlib-ng'
+description = """zlib data compression library for the next generation systems"""
+
+toolchain = {'name': 'GCCcore', 'version': '10.2.0'}
+
+source_urls = ['https://github.com/zlib-ng/zlib-ng/archive/']
+sources = ['%(version)s.tar.gz']
+checksums = ['eca3fe72aea7036c31d00ca120493923c4d5b99fe02e6d3322f7c88dbdcd0085']
+
+builddependencies = [
+    ('CMake', '3.18.4'),
+    ('binutils', '2.35'),
+]
+
+sanity_check_commands = [
+]
+
+sanity_check_paths = {
+    'files': [],
+    'dirs': ['lib/', 'include/'],
+}
```

Comment:
I think we have tests to run here, and no sanity check comamnd, and please specify the paths to the key libs and header instead of just dirs.